openstack/nova/values.yaml

```diff
@@ -752,6 +756,8 @@ mariadb_cell2:
     enabled: true
     oauth:
       client_id: "nova-cell2"
+  alerts:
+    support_group: compute-storage-api
 
 rabbitmq_cell2:
   name: nova-cell2
```

Comment:
Missed to add the `support_group` to the `alerts` section of this one. It should tell you when rendering the chart locally in for qa.